prow/config.yaml

```diff
@@ -310,9 +310,11 @@ postsubmits:
           - image: gcr.io/k8s-testimages/e2e-kubeadm:0.2
             args:
             - "--branch=$(PULL_REFS)"
-            - "--upload=gs://kubernetes-jenkins/logs"
-            - "--git-cache=/root/.cache/git"
             - "--clean"
+            - "--git-cache=/root/.cache/git"
+            - "--json"
+            - "--timeout=140"
```

Comment:
It's bootstrap timeout, it gives a bit overhead for bootstrap (20min) to checkout repo, activate service account, and upload logs, etc. KUBEKINS_TIMEOUT is still passed down to kubetest and remains the same.